spaceline-all-the-icons-segments.el

```diff
@@ -627,7 +627,7 @@ When FAMILY is provided, put `:family' property into face."
   (let* ((family (all-the-icons-icon-family icon))
          (height (if family 1.0 1.2))
          (icon-face `(:foreground ,(face-foreground face) :height ,(spaceline-all-the-icons--height height))))
-    (when family (plist-put icon-face :family family))
+    (when family (setq icon-face `(:family family)))
```

Comment:
This is missing the call to `(append` to the icon face, it also needs a comma before `family`